tests/test_suite_common.sh

```diff
@@ -335,17 +336,37 @@ EOF
 # - verify the templates generated by confd as a result.
 run_individual_test() {
     testdir=$1
+    testdir_save=$testdir
 
     # Populate Calico using calicoctl to load the input.yaml test data.
     echo "Populating calico with test data using calicoctl: " $testdir
     calicoctl apply -f /tests/mock_data/calicoctl/${testdir}/input.yaml
 
+    # Populate Kubernetes API with data if it exists for this test.
```

Comment:
Do we actually use this change for this PR? (the kubectl-input and kubectl-delete bits)

Reply:
Yes, because the mesh/static-routes test creates Kubernetes services, then later deletes those again.